frappe/utils/image.py

```diff
@@ -17,3 +17,26 @@ def resize_images(path, maxdim=700):
 					im.save(os.path.join(basepath, fname))
 
 					print("resized {0}".format(os.path.join(basepath, fname)))
+
+def strip_exif_data(content, content_type):
+	""" Strips exif from image files which support it.
+
+	Works by creating a new Image object which ignores exif by
+	default and then extracts the binary data back into content.
+
+	Returns: stripped image content
```

Comment:
```suggestion
	"""Strips EXIF metadata from image files which support it.

	Works by creating a new Image object which ignores EXIF by
	default and then extracts the binary data back into content.

	Returns: 
		Bytes: Stripped Image content
```

1. ~exif~ => EXIF
2. Mention format of returned value in docstring

This is just a suggestion. You may use your own formats, but make sure the docstring doesn't lie :P

Reply:
Sure

Reply:
made changes